pkgs/development/ocaml-modules/tcslib/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, ocamlPackages, buildOasisPackage, ounit, ocaml_extlib, num }:
+
+buildOasisPackage rec {
+  name = "tcslib";
```

Comment:
Common practice is to call this attribute “pname”, as it lacks the version information.